tests/unit/example.spec.js

```diff
@@ -0,0 +1,12 @@
+import { shallowMount } from "@vue/test-utils";
```

Comment:
I also tried to add **@testing-library/vue** but it has peer dependency to **vue-template-compiler**. And this compiler is only used for Vue 2 and seems to be deprecated for Vue 3. 

https://www.npmjs.com/package/vue-template-compiler
https://github.com/testing-library/vue-testing-library#installation

So using default **vue/test-utils**